project/model/src/main/java/vasilizas/bean/Admin.java

```diff
@@ -0,0 +1,10 @@
+package vasilizas.bean;
+
+public class Admin extends Person{
```

Comment:
Пробел перед скобкой
```suggestion
public class Admin extends Person {
```

Reply:
done